src/pedal/delay.rs

```diff
@@ -33,14 +33,12 @@ impl Pedal for Delay {
 
     fn process(&mut self, input: &[f32], output: &mut [f32]) -> Result<()> {
         ring_buffer::write_frame(&mut self.producer, input)?;
+        let samples: Vec<f32> = ring_buffer::read_frame(&mut self.consumer, output.len())?
```

Comment:
It seems like the name `samples` has become somewhat outdated - maybe `samples_at_level`?

Reply:
Could you elaborate on this?

Reply:
Just that `samples` would I'd think be the unmodified input samples, whereas these are now samples with their levels modified

Reply:
Ah, gotcha. I was thinking about the next PR where that difference no longer exists (since gain is a separate "pedal").